include/ui/connectionmanager.h

```diff
@@ -81,7 +81,7 @@ class ConnectionManager : public QObject {
   QAction* kill_locally_created_server_action_;
   QProcess* server_process_;
   ConnectionDialog* connection_dialog_;
-  bool shut_down_server_on_close_;
+  bool local_server_;
```

Comment:
It's bool, not a server object, so IMO it should be called `is_server_local_` or something similar.